apps/ui/src/components/FormSpaceWhitelabel.vue

```diff
@@ -127,7 +126,7 @@ onMounted(() => {
   <UiMessage
     v-if="isDisabled"
     type="info"
-    :learn-more-link="TURBO_URL"
+    :learn-more-link="{ name: 'space-pro' }"
     class="mb-4 max-w-[592px]"
   >
     Whitelabel features are only available for Turbo subscribers.
```

Comment:
We should also rename everywhere "Turbo" to "Snapshot Pro"
```suggestion
    Whitelabel features are only available for Snapshot Pro spaces.
```